### PR TITLE
fix signal test script

### PR DESCRIPTION
**Pull request by phoenixy1**
I had just kind of robotically filled out the test script for signal but it turns out requests and responses are accessed via different syntax, so the script didn't work. this fixes it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Follow these steps to quickly get started with the [Plaid API](https://plaid.com
...
...G9ydF90b2tlbiIsInZhbHVlIjoiIiwiZW5hYmxlZCI6dHJ1ZSwidHlwZSI6InRleHQiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjR9LHsia2V5IjoiZW52X3VybCIsInZhbHVlIjoic2FuZGJveC5wbGFpZC5jb20iLCJlbmFibGVkIjp0cnVlLCJ0eXBlIjoidGV4dCIsInNlc3Npb25WYWx1ZSI6InNhbmRib3gucGxhaWQuY29tIiwic2Vzc2lvbkluZGV4Ijo1fSx7ImtleSI6ImFjY291bnRfaWQiLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJ0ZXh0Iiwic2Vzc2lvblZhbHVlIjoiIiwic2Vzc2lvbkluZGV4Ijo2fSx7ImtleSI6InByb2Nlc3Nvcl90b2tlbiIsInZhbHVlIjoiIiwiZW5hYmxlZCI6dHJ1ZSwidHlwZSI6InRleHQiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjd9LHsia2V5IjoidHJhbnNmZXJfaWQiLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJ0ZXh0Iiwic2Vzc2lvblZhbHVlIjoiIiwic2Vzc2lvbkluZGV4Ijo4fSx7ImtleSI6InVzZXJfdG9rZW4iLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJ0ZXh0Iiwic2Vzc2lvblZhbHVlIjoiIiwic2Vzc2lvbkluZGV4Ijo5fSx7ImtleSI6InVzZXJfaWQiLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJ0ZXh0Iiwic2Vzc2lvblZhbHVlIjoiIiwic2Vzc2lvbkluZGV4IjoxMH0seyJrZXkiOiJwYXltZW50X2NvbnNlbnRfaWQiLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJhbnkiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjExfSx7ImtleSI6ImxpbmtfdG9rZW4iLCJ2YWx1ZSI6IiIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJhbnkiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjEyfV0=)
```
Comment (ToddKerpelman) [inline README.md:29]: Might be another caching issue, but when I tried this URL, I ended up with the old collection that still had the acccess_token typo.

**Review by ToddKerpelman — APPROVED** [review]
I was unable to test this (my sandbox environment doesn't include fraud_risk), but the code itself seems reasonable.

**Comment by phoenixy1** [comment]
@ToddKerpelman I tried regenerating the button and noticed an error message saying "Some error occurred while trying to update run button." I submitted a support ticket but I suspect Postman is slowly, via neglect, killing the static version of the "Run in Postman" button and that we may just be forced to move to the newer version of the button (the fork one) exclusively.

**Comment by Zazzberry** [comment]
What is this and please stop all payments if their were any discontinue this account it stealing from me


Do not use without a authorization from Krystal Dawn Williams 530274805
________________________________
From: Alex ***@***.***>
Sent: Wednesday, January 4, 2023 2:53 PM
To: plaid/plaid-postman ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] Re: [plaid/plaid-postman] fix signal test script (PR #138)

[External Email]

@ToddKerpelman<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FToddKerpelman&data=05%7C01%7Ckrystal.williams%40student.csn.edu%7C4dac1db5da8147c208ce08daeea69309%7Cdf35d5749fba48cea073737868d77a48%7C0%7C0%7C638084696455746451%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sA8uHQXCUvp%2B8AigWNc3KT9Q0XeCWv3V9N1qxrihsnE%3D&reserved=0> I tried regenerating the button and noticed an error message saying "Some error occurred while trying to update run button." I submitted a support ticket but I suspect Postman is slowly, via neglect, killing the static version of the "Run in Postman" button and that we may just be forced to move to the newer version of the button (the fork one) exclusively.

**Review by ToddKerpelman — APPROVED** [review]
Forking works! (My account doesn't seem to have access to fraud_risk, tho)

**Pull request merged**